static/js/admin/vue-permissions-editor/tests/unit/patronRoles.spec.js

```diff
@@ -1360,11 +1318,65 @@ describe('patronRoles.vue', () => {
     }
 
     function expectSaveButtonDisabled(expectDisabled = true) {
-        let disabledValue = wrapper.find('#is-submitting').attributes('disabled');
+        let disabledValue = wrapper.find('#is-submitting').attributes();
+
         if (expectDisabled) {
-            expect(disabledValue).toEqual('disabled');
+            expect(disabledValue).toHaveProperty('disabled');
         } else {
-            expect(disabledValue).not.toEqual('disabled');
+            expect(disabledValue).not.toHaveProperty('disabled');
         }
     }
+
+    function instantiateStore(resultObjects = []) {
+        return createStore({
```

Comment:
Aside form some default values, it seems like this blocks is basically the same as `store.js`, is there any way for this to use that rather than duplicating it? Particular the mutations.

Reply:
I thought about that, but was worried some of the tests could end up with weird data store states, since it would be shared across all tests, but never really tried it out.